package.json

```diff
@@ -71,26 +71,75 @@
     "zone.js": "~0.11.3"
   },
   "devDependencies": {
+    "diff": "^3.5.0",
+    "reflect-metadata": "^0.1.3",
+    "@material/animation": "^10.0.0",
+    "@material/auto-init": "^10.0.0",
```

Comment:
Shouldn't these match the version of `material-components-web`?

Reply:
Done, I didn't realize that the canary versions set different versions on the deps they had.